dex-it-common/src/main/scala/com/wavesplatform/dex/it/api/dex/WebSocket.scala

```diff
@@ -0,0 +1,35 @@
+package com.wavesplatform.dex.it.api.dex
+
+import java.util.concurrent.ConcurrentHashMap
+
+import akka.Done
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model.ws.{Message, TextMessage, WebSocketRequest}
+import akka.stream.Materializer
+import akka.stream.scaladsl.{Flow, Keep, Sink, Source}
+import com.wavesplatform.dex.domain.account.KeyPair
+import com.wavesplatform.dex.it.docker.DexContainer
+
+import scala.concurrent.Future
+
+class WebSocket(account: KeyPair, dex: DexContainer) {
+  implicit private val system: ActorSystem        = ActorSystem()
+  implicit private val materializer: Materializer = Materializer.matFromSystem(system)
```

Comment:
1. Should be outside, because some tests could require multiple WebSocket connections.
2. There is no cleanup for these resources

Consider move system and materializer to HasWebSocket and cleanup in HasWebSocket.afterAll

Reply:
Also HasWebSocket could have a method: `mkWebSocket(account, dex): WebSocket` (we have mkEntities)